packages/js/components/src/select-control/style.scss

```diff
@@ -126,7 +126,7 @@
 		top: 57px;
 		z-index: 10;
 		overflow-y: auto;
-		max-height: 350px;
+		max-height: 362px;
```

Comment:
We increased the max height of the select component to allow 9 items in the list without a scrollbar but only limited the height of the list to 4 items on a small screen for Core Profiler. Since this change will affect the other area that uses the select component, can we limit the list height to 4 items on a small screen for the select component directly? In this way, other areas that use the select component will also benefit from this change.

Reply:
Hey @chihsuan , that's a great idea thanks. I think I'll work on it as a follow-up so we can identify all the areas it's used and test those separately to this.